src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -199,6 +202,30 @@ export class CalcitePickList {
     });
   }
 
+  handleFilter = (event) => {
+    const filteredData = event.detail;
+    const values = filteredData.map((item) => item.value);
+    this.items.forEach((item) => {
+      if (values.indexOf(item.value) === -1) {
+        item.setAttribute("hidden", "");
```

Comment:
Is the filter component only for hiding or is it for matching elements too? If its for matching/highlighting then we probably shouldn't hide but set a class on it.

Reply:
the filter is provided the `filteredData` array. The pickList is controlling that any items not in that filteredList get hidden. We could hide them by whatever mechanism (adding a class vs hidden attribute).

Reply:
I just think it would be nice to componentize this logic so that it is consistent throughout any other components that want to filter items. Can be handled later.